util/config.go

```diff
@@ -0,0 +1,29 @@
+package util
```

Comment:
consider putting it in a `config` package

Reply:
Seems premature for one function. I think it's fine where it is.

Reply:
I put the constant in here, so now there are 2 things. `config.EnvPrefix` looks a lot nicer than `util.EnvPrefix` so I went ahead and moved it.